app/src/main/java/com/example/fragmentassg2/FragmentLogin.kt

```diff
@@ -4,64 +4,85 @@ import android.os.Bundle
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
+
+
+
         return inflater.inflate(R.layout.login_fragment,container,false)
-    }
 
+    }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        btnHindi?.setOnClickListener(this)
+        btnSkip?.setOnClickListener(this)
+        btnLogin?.setOnClickListener(this)
 
-        tvGood.setOnClickListener(){
-            Toast.makeText(context,"Good To See You Again",Toast.LENGTH_SHORT).show()
-        }
+        (requireActivity() as MainActivity).showBackButton(false)
 
-        tvDownload.setOnClickListener(){
-            Toast.makeText(context,"Download here",Toast.LENGTH_SHORT).show()
+        view.findViewById<TextInputEditText>(R.id.editPhoneIme).setOnEditorActionListener { v, actionId, event ->
+            return@setOnEditorActionListener when (actionId) {
+                EditorInfo.IME_ACTION_DONE -> {
+                    login()
+                    true
+                }
+                else -> false
+            }
         }
 
-        tvForgot.setOnClickListener(){
-            Toast.makeText(context,"Forgot PassWord",Toast.LENGTH_SHORT).show()
-        }
+    }
 
-        tvSSO.setOnClickListener(){
-            Toast.makeText(context,"Login with SSO",Toast.LENGTH_SHORT).show()
-        }
+    override fun onClick(v: View?) {
+        when(v){
 
-        tvSign.setOnClickListener(){
-            Toast.makeText(context,"Sign up",Toast.LENGTH_SHORT).show()
+            btnHindi->Toast.makeText(context,getString(R.string.lbl_btn_hindi),Toast.LENGTH_SHORT).show()
+            btnSkip->Toast.makeText(context,getString(R.string.lbl_btn_skip),Toast.LENGTH_SHORT).show()
+            btnLogin->{
+               login()
+            }
         }
+    }
 
-        btnGoogle.setOnClickListener(){
-            Toast.makeText(context,"Google login",Toast.LENGTH_SHORT).show()
-        }
+    private fun login(){
+        val number ="9953755955"
 
-        btnLogin.setOnClickListener(){
-            val number : String="9953755955"
-            if(editPhone.text.toString()==number){
-                val fragment=FragmentPassword()
-                val fragmentManager=activity!!.supportFragmentManager
-                val fragmentTransaction=fragmentManager.beginTransaction()
-                fragmentTransaction.setCustomAnimations(R.anim.enter_right_to_left,R.anim.exit_right_to_left,R.anim.enter_left_to_right,R.anim.exit_left_to_right)
-                fragmentTransaction.replace(R.id.root_layout,fragment)
-                fragmentTransaction.addToBackStack(null)
-                fragmentTransaction.commit()
-            }
-            else
-                Toast.makeText(context,"wrong",Toast.LENGTH_SHORT).show()
+        if(editPhone.editText?.text.toString()==number){
 
+            val fragmentManager=activity!!.supportFragmentManager
```

Comment:
@raghav32 make these variables global

and use requireActivity() instead of activity!!